.github/workflows/continuous-deployment.yml

```diff
@@ -0,0 +1,167 @@
+name: Continuous deployment
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  lint-python:
+    name: Lint Python code
+    runs-on: ubuntu-latest
+    steps:
+      - id: checkout
+        name: Checkout
+        uses: actions/checkout@v3
+
+      - id: setup-python
+        name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pipenv'
+
+      - id: install-pipenv
+        name: Install pipenv
+        run: curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python
+        shell: bash
+
+      - id: install-python-dependencies
+        name: Install Python dependencies
+        run: pipenv sync --dev
+        shell: bash
+
+      - id: lint
+        name: Lint Python code
+        run: pipenv run flake8 ./datawrapper --verbose
+        shell: bash
+
+  mypy-python:
+    name: Static-types check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - id: setup-python
+        name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pipenv'
+
+      - id: install-pipenv
+        name: Install pipenv
+        run: curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python
+        shell: bash
+
+      - id: install-python-dependencies
+        name: Install Python dependencies
+        run: pipenv sync --dev
+        shell: bash
+
+      - id: mypy
+        name: Run mypy
+        run: pipenv run mypy ./datawrapper --ignore-missing-imports --verbose
+        shell: bash
```

Comment:
`mypy` is a vestige of the original "hypermodern python package" template I used to create this repo back in the day ([link](https://github.com/cjolowicz/hypermodern-python)). 

I'm not sure that it's needed. Any opinions? 

Happy to leave this in for now and take this up in a different conversation.